runtime/darwinia/src/pallets/election_provider_multi_phase.rs

```diff
@@ -1,13 +1,17 @@
 // --- substrate ---
-use pallet_election_provider_multi_phase::{weights::SubstrateWeight, Config};
-use sp_runtime::{transaction_validity::TransactionPriority, Perbill};
+use pallet_election_provider_multi_phase::Config;
+use sp_runtime::{transaction_validity::TransactionPriority, PerU16, Perbill};
 // --- darwinia ---
 use crate::*;
+use weights::pallet_election_provider_multi_phase::WeightInfo;
 
 sp_npos_elections::generate_solution_type!(
 	#[compact]
-	pub struct NposCompactSolution16::<u32, u16, sp_runtime::PerU16>(16)
-	// -------------------- ^^ <NominatorIndex, ValidatorIndex, Accuracy>
+	pub struct NposCompactSolution16::<
+		VoterIndex = u32,
+		TargetIndex = u16,
+		Accuracy = PerU16,
+	>(16)
```

Comment:
Why crab is using ```NposCompactSolution24```, but darwinia is using ```NposCompactSolution16```

Reply:
Kusama: `NposCompactSolution24`
Polkadot: `NposCompactSolution16`

And I suggest increase Crab's validator count.